src/main/resources/component-biz.conf

```diff
@@ -1294,6 +1294,14 @@ storage {
                 # Specifies the read timeout for socket operations.
                 # Note that this needs to be higher than the idle timeout as otherwise the
                 readTimeout = 1815s
+
+                # Specifies the host name as exposed to the outside world.
+                # This information is used in the documentation at /kba/EAI5V
+                externalHost = ""
+
+                # Specifies the port as exposed to the outside world.
+                # This information is used in the documentation at /kba/EAI5V
+                externalPort = 22
```

Comment:
I would prefer not to set this value here. By default, the value from 'port' should be correct. externalPort should be used to overwrite it for documentational purpose only, when deployment setup requires it.

Reply:
Well, this port is what is exposed in Docker, and might be mapped differently in host OS or load-balancers / proxies.
But I'm OK in taking the default off.

Reply:
thx, sometimes there is more than one default ;-)